apps/explorer/src/app/components/nav/index.tsx

```diff
@@ -11,7 +11,7 @@ export const Nav = ({ menuOpen }: NavProps) => {
       <div
         className={`${
           menuOpen ? 'right-0 h-[100vh]' : 'right-[200vw] h-full'
-        } transition-[right] absolute top-0 w-full md:static md:border-r-1 bg-white dark:bg-black p-20`}
+        } transition-[right] absolute top-0 w-full md:static md:border-r-1 bg-white dark:bg-black p-20 z-50`}
```

Comment:
chore (not-sure-if-blocking): i think this might look a bit neater with `classnames`:

```
className={classNames('transition-[right] absolute top-0 w-full md:static md:border-r-1 bg-white dark:bg-black p-20 z-50', {
  'right-0 h-[100vh]': menuOpen,
  'right-[200vw] h-full': !menuOpen,
})
```

Reply:
Ach, you're right @notbot00. On it.